### ISSUE TITLE
Translation of iOS application

### ISSUE THREAD

**Issue by oleksandrlysenkov** [WeblateOrg/weblate#1698]
Hello,

I'm not sure how to translate iOS application, cos it has a separate file with translations for each User Interface file - http://prntscr.com/hgrr14 and also there is a separate file with manually added strings from the code - http://prntscr.com/hgrshv

In the filesystem it is organized this way for UI translations - http://prntscr.com/hgrrf4
And this way for that separate file with localizations - http://prntscr.com/hgrruy 

So basically I need to cover these: 
```
/Resources/*.lproj/Localizable.strings
/UI/*.lproj/Article.strings
/UI/*.lproj/Main.strings
```
Where * goes for a language code.

Thanks in advance for your help.

**Comment by nijel** [comment]
You need to create separate component for each of these filemasks.

**Comment by oleksandrlysenkov** [comment]
Thanks for the explanation.
